src/actions/action-creators.test.js

```diff
@@ -228,12 +228,19 @@ describe('Action creators', () => {
         min: -3,
         oversample: false,
         step: 1,
-        width: 300
+        width: 300,
+        mode: 'contain',
+        mathBounds: {
+          left: -10,
+          right: 10,
+          bottom: -10,
+          top: 10
+        }
       };
       initializeCalculator(desmosMock, calcContainerMock);
       return store.dispatch(actions.requestBurst(opts)).then(() => {
         // slide with min/max of -3/3 should dispatch addFrame 7 times
-        expect(store.getActions().length).toEqual(7);
+        expect(store.getActions().length).toEqual(1);
```

Comment:
Why is this now expected to equal to 1, rather than 7? If this has changed, the comment above doesn't match the code in place.